wazo_plugind/http.py

```diff
@@ -210,6 +221,18 @@ def add_resource(cls, api, *args, **kwargs):
         super().add_resource(api, *args, **kwargs)
 
 
+class StatusChecker(_BaseResource):
```

Comment:
API must be authenticated and inherite from `_AuthentificatedResource`

Reply:
Done. I have added the authentication and changed the test code accordingly.